### DISCUSSION TITLE
Github auto delete branch even though auto deleting is turned off for repository

### DISCUSSION THREAD

**Discussion by rinku-samanta-ts** [community/community#48273] [Pull Requests]
### Select Topic Area

Bug

### Body

I have a private repository for which I had auto delete branch after PR turned off, but it still deletes the branch after merge request is approved

<img width="1049" alt="image" src="https://user-images.githubusercontent.com/117336641/220634165-11603b00-1806-4fcb-ac34-74abdcc4cb0b.png">

**Comment by SmashedFrenzy16** [comment]
It might take some time to load. If it continues deleting the branches, contact GitHub support as it might be a bug. And if it does work afterwards and you want to restore the branches, you can.
